renovate.json

```diff
@@ -1,8 +1,72 @@
 {
     "extends": [
         ":onlyNpm",
-        "group:all"
+        "schedule:daily"
     ],
-    "enabledManagers": ["npm"],
-    "schedule": "on monday"
-}
+    "enabledManagers": [
+        "npm"
```

Comment:
Let's just do package.json and get it right for now -- we can cover requirements.txt later